CHANGELOG.md

```diff
@@ -5,16 +5,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### Added
 - Added `TM1` source class.
 - Added `TM1ToDF` task class.
 - Added `set_prefect_kv` parameter to `BigQueryToADLS` with `False` as a default. If there is a need to create new pair in KV Store the parameter can be changed to `True`.
-
+- Added `_rename_duplicated_fields` method to `SharepointListToDF` task class for finding and rename duplicated columns
+- Added new view type `agent_interaction_view_type` in `Genesys`source.
 ### Changed
 - Splitted test for Eurostat on source tests and task tests
-
-### Added
-- Added new view type `agent_interaction_view_type` in `Genesys`source.
+- Modified `SharepointList` source class: 
+  -> docstrings update
+- Modified `SharepointToADLS` flow class:
+  -> docstrings update
+  -> changed key_value_param: bool = False to prevent forced KV store append
+- Modified `SharepointListToADLS` flow class:
+  -> changed key_value_param: bool = False to prevent forced KV store append
+- Modified `SharepointList` source class:
```

Comment:
`Modified `SharepointList` source class:` is duplicated. You can use it once and fill with the changes